lib/mpl_toolkits/axisartist/angle_helper.py

```diff
@@ -56,10 +52,10 @@ def select_step_hour(dv):
     minsec_limits_ = [1.5, 2.5, 3.5, 4.5, 5.5, 8, 11, 14, 18, 25, 45]
     minsec_steps_  = [1,   2,   3,   4,   5,   6, 10, 12, 15, 20, 30]
 
-    minute_limits_ = A(minsec_limits_)*(1./60.)
+    minute_limits_ = np.array(minsec_limits_) / 60
```

Comment:
This looks like another case of old-school micro-optomizations like (`x*x` vs `x**2`), but this one still work!

```
In [3]: d = np.array(range(52))

In [4]: %timeit d / 60
936 ns ± 0.538 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)

In [5]: %timeit d * (1/ 60)
683 ns ± 1.43 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)

In [6]: d = np.array(range(50000))

In [7]: %timeit d / 60
102 µs ± 95.3 ns per loop (mean ± std. dev. of 7 runs, 10000 loops each)

In [8]: %timeit d * (1/ 60)
37.4 µs ± 554 ns per loop (mean ± std. dev. of 7 runs, 10000 loops each)


```

Reply:
I can repro, but I think axisartist/angle_helper is far enough from being on a critical code path to make it worth obfuscating...

Reply:
:+1: